matplotlib_scalebar/dimension.py

```diff
@@ -111,7 +112,11 @@ def __init__(self):
             latexrepr = None
             if prefix == "\u00b5" or prefix == "u":
                 latexrepr = _LATEX_MU + "m"
-            self.add_units(prefix + "m", factor, latexrepr)
+            if prefix == "Å" or prefix == "A" or prefix == "angstrom":
+                latexrepr = "Å"  # Directly add "Å" without appending "m"
```

Comment:
The correct LaTeX expression would be `\AA`

Reply:
> The correct LaTeX expression would be `\AA`

sure, this can be easily done. But sometimes the whole string needs to be in the math format like r"$\AA$" to display correctly, thus I put "Å" directly at the moment.